ee/src/main/java/org/jboss/as/ee/logging/EeLogger.java

```diff
@@ -1204,4 +1204,8 @@
     @LogMessage(level = INFO)
     @Message(id = 130, value = "%s hung task %s not cancelled")
     void hungTaskNotCancelled(String executorName, String taskName);
+
+    @Message(id = 131, value = "Failed to run scheduled task: %s")
+    RuntimeException failureWhileRunningTask(Object delegate, @Cause Exception e);
```

Comment:
This should probably throw an IllegalStateException, instead of a generic RuntimeException.